src/acquisition/flusurv/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.types import Date, Float, Integer, String
+
+Base = declarative_base()
+
+
+class FluSurv(Base):
+    """
+    SQLAlchemy model representing flusurve data.
+    """
+
+    __tablename__ = 'flusurv'
```

Comment:
suggestion: include constraints and indexes under their existing names

```suggestion
    __tablename__ = 'flusurv'
    __table_args__ = (
        UniqueConstraint("issue", "epiweek", "location", name="issue"),
        Index("release_date", "release_date"),
        Index("issue_2", "issue"),
        Index("epiweek", "epiweek"),
        Index("region", "location"),
        Index("lag", "lag"),
    )
```

From https://github.com/cmu-delphi/delphi-epidata/blob/4dc5f129704337a83bbd1ecd3bf2ad573d1e1e5f/src/ddl/fluview.sql#L386-L391

& referencing 
* https://docs.sqlalchemy.org/en/14/core/constraints.html#unique-constraint
* https://docs.sqlalchemy.org/en/14/orm/declarative_tables.html#declarative-table-configuration